### PR TITLE
Renaming payment methods to payment types #2

### PR DESCRIPTION
**Pull request by mingazhev**
Not the full renaming, just a part

### CODE REVIEW

File: Api/Services/Accommodations/Bookings/BookingExecution/Flows/BankCreditCardBookingFlow.cs [hunk 1]
```diff
@@ -46,16 +46,16 @@ public class BankCreditCardBookingFlow : IBankCreditCardBookingFlow
         public async Task<Result<string>> Register(AccommodationBookingRequest bookingRequest, AgentContext agentContext, string languageCode)
         {
             return await GetCachedAvailability(bookingRequest)
-                .Ensure(IsPaymentMethodAllowed, "Payment method is not allowed")
+                .Ensure(IsPaymentTypeAllowed, "Payment method is not allowed")
```
Comment (AlexeyShok) [inline Api/Services/Accommodations/Bookings/BookingExecution/Flows/BankCreditCardBookingFlow.cs:49]: May be: "Payment type is not allowed"?

File: Api/Services/Accommodations/Bookings/BookingExecution/Flows/FinancialAccountBookingFlow.cs [hunk 2]
```diff
@@ -40,7 +40,7 @@ public class FinancialAccountBookingFlow : IFinancialAccountBookingFlow
             AgentContext agentContext, string languageCode, string clientIp)
         {
             return GetCachedAvailability(bookingRequest)
-                .Ensure(IsPaymentMethodAllowed, "Payment method is not allowed")
+                .Ensure(IsPaymentTypeAllowed, "Payment method is not allowed")
```
Comment (AlexeyShok) [inline Api/Services/Accommodations/Bookings/BookingExecution/Flows/FinancialAccountBookingFlow.cs:43]: May be: "Payment type is not allowed"?

File: Api/Services/Accommodations/Bookings/BookingExecution/Flows/OfflinePaymentBookingFlow.cs [hunk 3]
```diff
@@ -35,7 +35,7 @@ public class OfflinePaymentBookingFlow : IOfflinePaymentBookingFlow
             AgentContext agentContext, string languageCode, string clientIp)
         {
             return GetCachedAvailability(bookingRequest)
-                .Ensure(IsPaymentMethodAllowed, "Payment method is not allowed")
+                .Ensure(IsPaymentTypeAllowed, "Payment method is not allowed")
```
Comment (AlexeyShok) [inline Api/Services/Accommodations/Bookings/BookingExecution/Flows/OfflinePaymentBookingFlow.cs:38]: May be: "Payment type is not allowed"?

**Review by AlexeyShok — APPROVED** [review]

File: Api/Services/Accommodations/Bookings/BookingExecution/Flows/BankCreditCardBookingFlow.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Bookings/BookingExecution/Flows/BankCreditCardBookingFlow.cs]: Fixed

File: Api/Services/Accommodations/Bookings/BookingExecution/Flows/FinancialAccountBookingFlow.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Bookings/BookingExecution/Flows/FinancialAccountBookingFlow.cs]: Fixed

File: Api/Services/Accommodations/Bookings/BookingExecution/Flows/OfflinePaymentBookingFlow.cs (hunk 3 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Bookings/BookingExecution/Flows/OfflinePaymentBookingFlow.cs]: Fixed

**Review by kirillta — APPROVED** [review]

**Pull request merged**